platformcomponents/desktop/participant-list.json

```diff
@@ -0,0 +1,16 @@
+{
+  "participantList": {
+    "comment": "controls at participantList panel, mute/unmute, etc.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=9332%3A2007",
+    "audioMuted": {
+      "#normal": {
+        "background": "@theme-indicator-attention"
```

Comment:
We do not need the 'background' layer. You can keep it in:
```
"audioMuted":{
     #normal: "@theme-indicator-attention"
}
```
Same for audioUnmuted

Reply:
@nataliadelmar ，I am not sure if the sub-layer is in need when only a color-theme will be applied this component. I am interested when we should add the sub-layer (such as background and hover, etc)